sys/arm64/arm64/trap.c

```diff
@@ -91,8 +91,9 @@ typedef void (abort_handler)(struct thread *, struct trapframe *, uint64_t,
     uint64_t, int);
 
 static abort_handler align_abort;
+#if __has_feature(capabilities)
 static abort_handler cap_abort;
-static abort_handler cap_mmu_abort;
+#endif
```

Comment:
Might be inclined to separate this out into a separate block, although it'll probably stay as just the one so fine as is.

Reply:
Yes, I don't expect we will add any more CHERI-specific abort handlers at this point.